test/conf/slave_store_2.conf

```diff
@@ -1,8 +1,17 @@
 store {
+    # ip
+    ip = "127.0.0.1"
     # Here is the server port that Dst-Server listens on.
-    listeningPort = 18091
-    # Whether this is the master
-    isMaster = false
+    listeningPort = 8093
```

Comment:
how about merge these to `address`?

Maybe it should be made in a serprated PR.